index.html

```diff
@@ -196,7 +196,7 @@
               const accessibleColorTransform = { // this way you don't have to change everything in the google spreadsheet
                 "#fc03df": "#2c7bb6",
                 "#03bafc": "#abd9e9",
-                "#9f48ea": "#fdae61",
+                "#9F48EA": "#fdae61",
```

Comment:
Should we do this or convert to lower case when we use this mapping?

Reply:
Thanks for catching. I think it makes sense to use `toLowerCase()` when using the mapping to make sure everything is standard.